src/app/page/UserProfile/UserProfile.js

```diff
@@ -106,78 +138,75 @@ const UserProfile = ({ history, ...props }) => {
 
   async function errorHandler(error) {
```

Comment:
We should extract this out later, so that we can reuse this merge logic at other places: i.e. authenticate phone before volunteering.